plugins/GSdx/GSRenderer.cpp

```diff
@@ -83,6 +83,27 @@ void GSRenderer::ResetDevice()
     if(m_dev) m_dev->Reset(1, 1);
 }
 
+// Mizuiro constantly alters it's DH value on the DISP register from 464 to 480
+// This causes the screen to shake vertically due to the frequent change in DISP register value
+// This doesn't happen in the PS2, the assumption is that the GS has a delay before applying
+// a new DH value to the output circuit. The following function handles such delay.
```

Comment:
You mean that DH value is overwritten before it is use, isn't it ?

What is "frequent" change for you ? Before Vsync ?